readthedocs/settings/base.py

```diff
@@ -242,6 +244,14 @@ def USE_PROMOS(self):  # noqa
 
     CELERY_DEFAULT_QUEUE = 'celery'
 
+    CELERYBEAT_SCHEDULE = {
+        'weekly-clear-persistent-messages': {
+            'task': 'readthedocs.core.tasks.clear_persistent_messages',
+            'schedule': crontab(hour=23, minute=59, day_of_week=6),
```

Comment:
Any reason this can't run on a more frequent schedule? I'd like this to run in my development instance as well, and I'm sure not going to wait up for it to run :)

Running on a `hour='*/3'` or something might make more sense.